include/rocksdb/utilities/transaction_db.h

```diff
@@ -104,6 +104,11 @@ class RangeLockManagerHandle : public LockManagerHandle {
     // Number of times lock escalation was triggered (for all column families)
     uint64_t escalation_count;
 
+    // Number of times lock acquisition had to wait for a conflicting lock
+    // to be released. This counts both successful waits (where the desired
+    // lock was acquired) and waits that timed out or got other error.
+    uint64_t lock_wait_count;
```

Comment:
New to range locking code. Will it be more informative if we track successful waits and time-outs in two separate counters?